packages/docs/src/routes/docs/(qwikcity)/api/index.mdx

```diff
@@ -276,6 +276,19 @@ The `routeAction$()` function is used to declare a new Server Action in the give
 
 Please refer to the [Server Actions](/docs/(qwikcity)/action/index.mdx) section for more information.
 
+## List all routes with qwikCityPlan.routes
+
+The `qwikCityPlan.routes` property is used to retrieve the list of the available routes of your application.
+
+```tsx
+import qwikCityPlan from '@qwik-city-plan';
+
+const routes = qwikCityPlan.routes;
+for (const route of routes) {
+	// your logic here
+}
+```
```

Comment:
nice I didn't know about that. Where can this be used? What's the difference with useMenu? Should we add a line or two to explain when to use which?